docs/conf.py

```diff
@@ -27,9 +27,17 @@
     'sphinx.ext.autosummary',
     'sphinx.ext.mathjax',
     'sphinx.ext.viewcode',
-    'sphinx_rtd_theme',
+#    'sphinx_rtd_theme',
```

Comment:
This needs to be commented out to reproduce the error.

Or to state it differently: Because of the RTD injection, adding `'sphinx_rtd_theme'` to `extensions` is necessary.. otherwise `setup(app)` is not called.